pkg/template/template_test.go

```diff
@@ -17,19 +17,19 @@ import (
 var EFS embed.FS
 
 //go:embed testdata/host/*
-var hostFS embed.FS
+var HostFS embed.FS
```

Comment:
this exported variable seems not used outside this file

Reply:
good point, see https://github.com/codeready-toolchain/toolchain-common/pull/373/commits/0501aa9c6ce1538834ba6c8b8b159174e01d1527